deepee/wrapper.py

```diff
@@ -282,6 +270,12 @@ def parameters(self):
             " Please use <Instance>.model.parameters()."
         )
 
+    @property
+    def current_epsilon(self):
+        if self.watchdog is None:
+            raise ValueError("No watchdog to calculate epsilon")
+        return self.watchdog.calc_epsilon(self._steps_taken)
+
 
 class PerSampleGradientWrapper(nn.Module):
```

Comment:
We should fix this class to be on par with the changes to the PrivacyWrapper, especially w.r.t the checks for calling the methods in order